### ISSUE TITLE
Playback fails inexplicably

### ISSUE THREAD

**Issue by xykrr** [DeaDBeeF-Player/deadbeef#2458]
Deadbeef version: 1.8.4
OS: Artix

Whenever I try to play a track, I get this error:
```
Failed to play track: /home/%USER%/Music/Stax Jazz Box 2CD/Stax Jazz CD 1/13 - Detour Ahead - Stan Getz.flac
starting deadbeef 1.8.4
server_start
searching for GUI plugins in /home/%USER&/.local/lib64/deadbeef
searching for GUI plugins in /home/%USER%/.local/lib/deadbeef
load_plugin_dir /home/%USER%/.local/lib/deadbeef: scandir found 3 files
searching for GUI plugins in /usr/lib/deadbeef
load_plugin_dir /usr/lib/deadbeef: scandir found 54 files
found gui plugin ddb_gui_GTK2.so
added GTK2 gui plugin
found gui plugin ddb_gui_GTK3.so
added GTK3 gui plugin
load gui plugin
checking GUI plugin: GTK2
found selected GUI plugin: GTK2
loading plugin /usr/lib/deadbeef/ddb_gui_GTK2.so
loading plugins from /home/%USER%/.local/lib64/deadbeef
loading plugins from /home/%USER%/.local/lib/deadbeef
load_plugin_dir /home/%USER%/.local/lib/deadbeef: scandir found 3 files
loading plugins from /usr/lib/deadbeef
load_plugin_dir /usr/lib/deadbeef: scandir found 54 files
loading plugin /usr/lib/deadbeef/aac.so
loading plugin /usr/lib/deadbeef/adplug.so
loading plugin /usr/lib/deadbeef/alac.so
loading plugin /usr/lib/deadbeef/alsa.so
loading plugin /usr/lib/deadbeef/artwork.so
loading plugin /usr/lib/deadbeef/cdda.so
loading plugin /usr/lib/deadbeef/converter.so
loading plugin /usr/lib/deadbeef/converter_gtk2.so
loading plugin /usr/lib/deadbeef/converter_gtk3.so
loading plugin /usr/lib/deadbeef/dca.so
loading plugin /usr/lib/deadbeef/ddb_dumb.so
loading plugin /usr/lib/deadbeef/ddb_mono2stereo.so
loading plugin /usr/lib/deadbeef/ddb_shn.so
loading plugin /usr/lib/deadbeef/ddb_soundtouch.so
loading plugin /usr/lib/deadbeef/dsp_libsrc.so
loading plugin /usr/lib/deadbeef/ffap.so
loading plugin /usr/lib/deadbeef/ffmpeg.so
loading plugin /usr/lib/deadbeef/flac.so
loading plugin /usr/lib/deadbeef/gme.so
loading plugin /usr/lib/deadbeef/hotkeys.so
loading plugin /usr/lib/deadbeef/in_sc68.so
loading plugin /usr/lib/deadbeef/lastfm.so
loading plugin /usr/lib/deadbeef/m3u.so
loading plugin /usr/lib/deadbeef/mms.so
loading plugin /usr/lib/deadbeef/mp3.so
loading plugin /usr/lib/deadbeef/musepack.so
loading plugin /usr/lib/deadbeef/notify.so
loading plugin /usr/lib/deadbeef/nullout.so
loading plugin /usr/lib/deadbeef/oss.so
loading plugin /usr/lib/deadbeef/pltbrowser_gtk2.so
loading plugin /usr/lib/deadbeef/pltbrowser_gtk3.so
loading plugin /usr/lib/deadbeef/psf.so
loading plugin /usr/lib/deadbeef/pulse.so
loading plugin /usr/lib/deadbeef/rg_scanner.so
loading plugin /usr/lib/deadbeef/shellexec.so
loading plugin /usr/lib/deadbeef/shellexecui_gtk2.so
loading plugin /usr/lib/deadbeef/shellexecui_gtk3.so
loading plugin /usr/lib/deadbeef/sid.so
loading plugin /usr/lib/deadbeef/sndfile.so
loading plugin /usr/lib/deadbeef/supereq.so
loading plugin /usr/lib/deadbeef/tta.so
loading plugin /usr/lib/deadbeef/vfs_curl.so
loading plugin /usr/lib/deadbeef/vfs_zip.so
loading plugin /usr/lib/deadbeef/vorbis.so
loading plugin /usr/lib/deadbeef/vtx.so
loading plugin /usr/lib/deadbeef/wavpack.so
loading plugin /usr/lib/deadbeef/wildmidi.so
loading plugin /usr/lib/deadbeef/wma.so
starting plugin GTK2 user interface
starting plugin AAC player
starting plugin Adplug player
starting plugin ALAC player
starting plugin ALSA output plugin
starting plugin Album Artwork
starting plugin Audio CD player
starting plugin Converter
starting plugin Converter GTK2 UI
starting plugin Converter GTK3 UI
starting plugin dts decoder
starting plugin DUMB module player
starting plugin Mono to stereo
starting plugin Shorten player
starting plugin Soundtouch
starting plugin Resampler (Secret Rabbit Code)
starting plugin Monkey's Audio (APE) decoder
starting plugin FLAC decoder
starting plugin Game-Music-Emu player
starting plugin Hotkey manager
starting plugin SC68 player (Atari ST SNDH YM2149)
starting plugin last.fm scrobbler
starting plugin M3U and PLS support
starting plugin mms vfs
starting plugin MP3 player
starting plugin MusePack decoder
starting plugin OSD Notify
starting plugin Null output plugin
starting plugin OSS output plugin
starting plugin Playlist browser GTK2
starting plugin Playlist browser GTK3
starting plugin PSF player using Audio Overload SDK
starting plugin PulseAudio output plugin
starting plugin ReplayGain Scanner
starting plugin Shell commands
starting plugin Shellexec GTK2 UI
starting plugin Shellexec GTK3 UI
starting plugin SID player
starting plugin WAV/PCM player
starting plugin SuperEQ
starting plugin tta decoder
starting plugin cURL vfs
starting plugin ZIP vfs
starting plugin Ogg Vorbis decoder
starting plugin VTX player
starting plugin WavPack decoder
starting plugin WildMidi player
starting plugin WMA player
starting plugin stdio vfs
starting plugin FFMPEG audio player
selected output plugin: ALSA output plugin
plugin Converter GTK3 UI failed to connect to dependencies, deactivated.
plugin Playlist browser GTK3 failed to connect to dependencies, deactivated.
plugin Shellexec GTK3 UI failed to connect to dependencies, deactivated.
resume: track 16 pos 0.000000 playlist 4
Failed to play track: /home/%USER%/Music/Stax Jazz Box 2CD/Stax Jazz CD 2/01 - Everytime We Say Goodbye - Ella Fitzgerald.flac
```

From that log alone, I can't make sense of what's causing the issue. I've tried GTK2 GTK3 interfaces. Always the same issue except it'll say the GTK2 counterparts failed when I'm running GTK3. Yet, it's not even likely that's what's causing playback to fail anyway.
The culprit likely lies within my DeaDBeeF configs/playlists/etc having been imported from a previous Ubuntu based installation, where DeaDBeeF worked fine. Somehow things were corrupted in the process.
Any ideas or next steps in troubleshooting? I'm also not sure I've figured out where all the configuration files lie- I've just been going by the log. So, I'd appreciate explicit naming of what things to try modifying or deleting.

**Comment by Alexey-Yakovenko** [comment]
Deadbeef playlists use absolute paths, so if you changed your Music folder location -- you need to re-create the playlists.
Looks like a duplicate of #1798

**Comment by xykrr** [comment]
I'm aware that DeaDBeeF uses absolute paths- that's what I first assumed was the issue. That is not what's happening. DeaDBeeF fails to play files in new playlists with new paths.

**Comment by Alexey-Yakovenko** [comment]
All configuration is stored in `~/.config/deadbeef`. I can't see any issue from the log, except that it can't play some flac files for some reason. Are you sure that the files are accessible / have correct permissions?

**Comment by xykrr** [comment]
Wow, I just figured out what was precisely wrong- I feel like a dunce!
I tried removing my previous config so DeaDBeeF would make a fresh one. The fresh config worked fine. While I was somewhat ready to start over with configuring stuff, I was still curious what was wrong with my previous config. So, I deleted the new config and added back in my old one. I then checked the preferences, only to find I had ALSA set to no specific device- I forgot to rebind my audio device since I used DeaDBeeF last on my previous OS! Doh!! I just set audio to PulseAudio and everything worked fine with my old config- I'll just have to reinstall some plugins that didn't carry over with it.
Thanks anyway
